src/rime/gear/chord_composer.cc

```diff
@@ -27,6 +27,8 @@ ChordComposer::ChordComposer(const Ticket& ticket) : Processor(ticket) {
     config->GetBool("chord_composer/use_shift", &use_shift_);
     config->GetBool("chord_composer/use_super", &use_super_);
     config->GetBool("chord_composer/use_caps", &use_caps_);
+    config->GetBool("chord_composer/finish_when_release",
```

Comment:
`finish_when_release` isn't helpful in distinguishing the two behaviours:
both finish chords on key release; the difference is that the new behaviour finishes a chord on the *first* key release instead of the last / all pressed keys.

if you don't mind being verbose, `finish_chord_on_first_key_release` is by no means ambiguous.